LamaObject object
DescriptorIdentifier descriptor
MapAction action
---
LamaObject[] objects
DescriptorIdentifier[] descriptors
